src/Commands/BannedKeywordsBag.ts

```diff
@@ -37,6 +37,7 @@ export class BannedKeywordsBag {
             byte: true,
             case: true,
             catch: true,
+            ceiling: true,
```

Comment:
Why is this a banned keyword? Is there a language that has `ceiling` as a keyword? If not, we can leave it out of here.

Reply:
Ah! Sorry about that, that was leftover from when I was still tinkering around with the project.  Removed in 4730206d82c3ecfb9b5605e5af2b6fcbe5aca8eb.  Thanks for the feedback!